layers/+tools/chrome/config.el

```diff
@@ -0,0 +1,2 @@
+(defvar chrome-exec-path "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
```

Comment:
A better solution would be to have the default be `nil` and then when it is `nil` search `PATH` for `google-chrome`, `google-chrome-stable`, `chromium`, and whatever you do to find programs on Mac.